portfolio.html

```diff
@@ -43,65 +43,57 @@
+          <a href="/" class="dwyl-dark-gray link">
             <img src="./img/common/dwyl-heart-only-logo.png" alt="dwyl heart logo" class="w1-plus"/>
             dwyl
           </a>
         </li>
-        <li class="ph4-l ph5-xl ph5 pt5 pb2 tl dn-l"><a href="/" class="white link topnav">Home</a></li>
-        <li class="ph4-l ph5-xl ph5 di-l pv2 tl pv0-l"><a href="/portfolio" class="white link topnav">Portfolio</a></li>
-        <li class="ph4-l ph5-xl ph5 di-l pv2 tl pv0-l"><a href="/values" class="white link topnav">Values</a></li>
-        <li class="ph4-l ph5-xl ph5 di-l pv2 tl pv0-l"><a href="/team" class="white link topnav">Team</a></li>
-        <li class="ph4-l ph5-xl ph5 dib-l pv2 tl pv0-l"><a href="/time-app" class="white link topnav">App
-          <span class="dwyl-bg-gold pa1 br2 f7">BETA</span></a></li>
-        <li class="ph4-l ph5-xl ph5 di-l pv2 tl pv0-l"><a href="/blog" class="dwyl-mint link topnav">Blog</a></li>
-        <li class="ph4-l ph5-xl ph5 di-l pt2 pb3 tl pv0-l"><a href="#contact" id="contact-link" class="white link topnav">Contact</a></li>
+        <li class="ph4 pt5 pb2 tl dn-l"><a href="/" class="dwyl-dark-gray link">Home</a></li>
+        <li class="ph4 di-l pv2 tl pv0-l"><a href="/portfolio" class="dwyl-dark-gray link">Portfolio</a></li>
+        <li class="ph4 di-l pv2 tl pv0-l"><a href="/values" class="dwyl-dark-gray link">Values</a></li>
+        <li class="ph4 di-l pv2 tl pv0-l"><a href="/team" class="dwyl-dark-gray link">Team</a></li>
+        <li class="ph4 dib-l pv2 tl pv0-l"><a href="/time-app" class="dwyl-dark-gray link">App
+          <span class="dwyl-bg-yellow fw3 pa1 br2 f7 shadow-6">BETA</span></a></li>
+        <li class="ph4 di-l pv2 tl pv0-l"><a href="/blog" class="dwyl-mint link">Blog</a></li>
+        <li class="ph4 di-l pt2 pb3 tl pv0-l"><a href="#contact" id="contact-link" class="dwyl-dark-gray link">Contact</a></li>
       </ul>
     </nav>
 
     <section class="bg-white pt6 pb4">
-      <section class="mw9 center ph3">
-        <h1 class="pl3 pr3 f2 w-100 tc tl-ns dwyl-dark-gray">Portfolio</h1>
+      <section class="mw8 center pt5 ph3">
+        <h1 class="pl3 pr3 f1 w-100 tc tl-ns dwyl-dark-gray">Portfolio</h1>
         <p class="pl3 pr3 f4 b ph0-m dwyl-dark-gray pb3">We're currently reviewing the content of the case studies below and we'll be adding them to the site soon.</p>
-        <div class="w-10-ns pt2 ph2 pb3 dib-ns dtc-ns v-top tc center">
-          <a href="/portfolio" class="link dwyl-dark-gray">
-            <div class="pa2 bg-white br2 h5 h6-l shadow-2">
-              <p class="pa2 bb b--black pb3 tl-ns montserrat f6">More than 250,000 artefacts, made accessible
-                and searchable!</p>
-              <img src="./img/common/science-museum-logo.jpg" alt="Science Museum logo" class="w-70 w-75-l pt3 pt2-l"/>
-            </div>
+        <div class="dt-m dt-l dt--fixed-m dt--fixed-l mb3 ph3 ph0-ns" style="border-spacing: 1rem 0px;">
```

Comment:
These new cards are looking great :tada:

Shall we open an issue about the weird lack of consistency with the github cards directly below it?